src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphSubmissionDAO.scala

```diff
@@ -88,7 +102,8 @@ class GraphSubmissionDAO extends SubmissionDAO with GraphDAO {
 
   private def fromVertex(workspaceNamespace: String, workspaceName: String, vertex: Vertex): Submission = {
```

Comment:
lazy vals?  or is this the case where if we tried to get after the DAO it would throw that orient thread local exception?